src/assets/src/components/preferences.tsx

```diff
@@ -111,11 +185,49 @@ function PreferencesEditor(props: PreferencesEditorProps) {
         <div>
             <h1>View/Update Preferences</h1>
             {alertBlock}
-            <Form onSubmit={validateAndSubmit}>
-                <p>Enter a phone number in order to opt in to SMS notifications.</p>
-                <FormGroup controlId='phone' className="mb-3">
-                    <Form.Label>Phone Number</Form.Label>
-                    {phoneInput}
+            <h2>Contact Information</h2>
+            {phoneUpdateStatus === "" &&
+                <Form>
+                    <p>Enter a phone number in order to opt in to SMS notifications.</p>
+                    <FormGroup controlId='phone' className="mb-3">
+                        <Form.Label>Phone Number</Form.Label>
+                        {phoneInput}
+                    </FormGroup>
+
+                    {!sendingCode && changedPhoneNumber ? <Button variant="secondary" type="submit" disabled={props.disabled} onClick={getOneTimePassword}>Get a one-time passcode for verification</Button>
```

Comment:
Could you please update the label to "Obtain a one-time phone verification code."? Thanks!